Pipfile

```diff
@@ -13,6 +13,7 @@ gunicorn = "==19.9.0"
 fabric = "==2.4.0"
 social-auth-app-django = "==3.1.0"
 django-autocomplete-light = "==3.5.1"
+livereload = "*"
```

Comment:
Let's remove all Pipfile changes -- we typically want to have one dedicated PR to tackle dependency updates.